prague-postgres-meetup-september-2024.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Prague PostgreSQL Meetup on September 30th'
+description: 'The fifth Prague PostgreSQL Meetup of 2024 will take place on September 30th!'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/enter-image-here.png
+  alt: Prague PostgreSQL Meetup on September 30th
+author: Gulcin Yildirim Jelinek
+authorEmail: gulcin@xata.io
+date: 09-25-2024
+tags:
+  ['postgres', 'postgresql', 'prague-postgres-meetup', 'prague', 'praha', 'meetup', 'event', 'community', 'september']
+published: true
+slug: prague-postgres-meetup-september-2024
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/enter-image-here.png
+---
+
+## Prague PostgreSQL Meetup on September 30th
+
+Besides working at Xata, I am also the main organizer of the monthly [Prague PostgreSQL Meetup](https://www.meetup.com/prague-postgresql-meetup/).
+I am thrilled to invite you to the fifth meetup of 2024, taking place on Monday, September 30th, at 18:00.
+If you are in the area, please join us for an evening of engaging talks, networking, and refreshments.
+
+### A bit of history
+
+I visited Prague for the first time in 2015 and I instantly fell in love with the city.
+I am happy to say, I was lucky to move here two years later and I have been living in this beautiful city since then!
+
+I am also known to be a bit of a Postgres enthusiast, and I have been using Postgres in one form or another since 2012.
+When I started working at 2ndQuadrant in 2015, I became friends with Tomas Vondra, who also happens to be a Postgres enthusiast.
```

Comment:
Some links to 2ndQuadrant (if any EDB relevant ones exist) and a Tomas Vondra LinkedIn might be useful